packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1397,14 +1403,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             return;
         }
 
-        // If switching to manual mode add a warning trace indicating the underlying loader does not support
-        // this feature yet. Can remove in 0.9.
-        if (!this.deltaSender && mode === FlushMode.Manual) {
```

Comment:
I believe you can remove this because this.deltaSender can never be undefined. If so, can you please fix its type to reflect that?